packages/core/src/internal/services/i18n.service.ts

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export interface I18nStrings {
+  [key: string]: { [key: string]: string | undefined };
+
+  alert: {
+    closeButtonAriaLabel: string;
+    loading: string;
+    info: string;
+    success: string;
+    warning: string;
+    danger: string;
+  };
+  datagrid: {
```

Comment:
Are we still planning on  removing these and only adding them as the component is added?

Reply:
Yup, I will remove all the extra ones that aren't currently used in core.